CMakeTests.txt

```diff
@@ -0,0 +1,228 @@
+# Copyright 2017-2022 Uber Technologies, Inc.
```

Comment:
You might want to use the `cmake` extension for this file instead of `txt`. I think that will trigger Github's code highlighting to treat this as CMake.

Reply:
I avoided it, though, because the docs implied that it will switch to a CMake module mode instead of simple C-style include insertion of the contents of the file. I will give it a try, but this may not be possible.

Reply:
Looks like it did work.